packages/testing/src/chai-retry-plugin/helpers.ts

```diff
@@ -0,0 +1,45 @@
+import Chai from 'chai';
+import { timeout as addTimeoutToPromise, sleep } from 'promise-assist';
```

Comment:
we can leave is as `timeout` now. Or even better in my opinion - call it `timeoutPromise`.

Reply:
@yurii-ve fixed